rewrite-gradle/src/main/java/org/openrewrite/gradle/trait/Traits.java

```diff
@@ -26,4 +26,8 @@ public static GradleDependency.Matcher gradleDependency() {
     public static JvmTestSuite.Matcher jvmTestSuite() {
         return new JvmTestSuite.Matcher();
     }
+
+    public static GradlePlugin.Matcher plugin() {
+        return new GradlePlugin.Matcher();
+    }
```

Comment:
As a quick FYI we're moving away from `Trait` classes as they can lead to import conflicts:
- https://github.com/openrewrite/rewrite/pull/5613
- https://github.com/openrewrite/rewrite-rewrite/pull/16

The suggested replacement is just an inlined `new GradlePlugin.Matcher()`. Likely best to then not add this here & adjust the tests.

Reply:
No problem! I'll get this cleaned up tomorrow as a part of that discussions outcome (I was a small part of that discussion as well. 😉).